guides/plugins/plugins/api/multi-inventory.md

```diff
@@ -0,0 +1,147 @@
+# Multi-Inventory
+
+## Pre-requisites and setup
+
+### Commercial Plugin
+
+As Multi-Inventory is part of the Commercial plugin, it requires an existing Shopware 6 installation and the activated Commercial plugin on top. The plugin can be installed following these [install instructions](../../../../guides/plugins/plugins/plugin-base-guide#install-your-plugin). In addition, the Multi-Inventory feature requires [Shopware Beyond](https://docs.shopware.com/en/shopware-6-en/features/shopware-beyond).
```

Comment:
I would keep it this way "Multi-Inventory is part of the Commercial plugin", since it really is just a specific feature inside the plugin, along with other unrelated features. Applied the rest. 👍🏻